web-client/src/presenter/computeds/formattedTrialSessionDetails.ts

```diff
@@ -88,15 +88,22 @@ export const formattedTrialSessionDetails = (
 
     const user = applicationContext.getCurrentUser();
     const isChambersUser = user.role === USER_ROLES.chambers;
-
     const trialDateInFuture = trialDateFormatted > nowDateFormatted;
+    const docketClerkCanEditCheck = sessionType => {
+      const editableSessionTypes = ['Special', 'Motion/Hearing'];
+      return editableSessionTypes.includes(sessionType);
+    };
 
     canDelete = trialDateInFuture && !formattedTrialSession.isCalendared;
     canEdit =
       trialDateInFuture &&
       formattedTrialSession.sessionStatus !== SESSION_STATUS_GROUPS.closed &&
       !isChambersUser;
 
+    if (user.role === USER_ROLES.docketClerk && canEdit) {
```

Comment:
Kept, it simple here for now but this could always change if we find that it is necessary to have additional checks for other roles